x-pack/plugins/cases/public/components/user_actions/comment/user.tsx

```diff
@@ -7,6 +7,8 @@
 
 import React from 'react';
 import classNames from 'classnames';
+import styled from 'styled-components';
```

Comment:
[It's now recommended to use `@emotion` for this.](https://github.com/elastic/kibana/issues/115326)

Reply:
Actually there was an issue with using `@emotion` after this [PR](https://github.com/elastic/kibana/pull/146212) was merged. 

Before this PR `cases code `was only being built with `style-components` babel plugin, not the emotion babel plugin.

Which caused incorrect behaviour of styles after merging above PR.

So it was decided to migrate the small number of components using emotion to styled-components, and then move all of cases to emotion and remove cases from 

https://github.com/elastic/kibana/blob/afb09ccf8a61d610e8e3d8beb2c80f413f1b33c5/packages/kbn-babel-preset/webpack_preset.js#L42-L65

 @cnasikas made this change in this PR:  #148065 
 
 So I think it would be better to migrate all styled components to `@emotion` and make necessary changes in a separate PR. What do you think? @adcoelho

Reply:
Sounds good, didn't know about that 👌